src/localization/messages_tl.js

```diff
@@ -0,0 +1,11 @@
+/*
+ * Translated default messages for the jQuery validation plugin.
+ * Locale: TL (Tagalog; Tagalog)
```

Comment:
Should "Tagalog" be written only once at this spot (since AFAIK it's written the same way in both English and Tagalog)?